From 81275780fbc36fc2f76304672ac6d329c1813fb4 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 09 Nov 2020 13:20:51 +0200
Subject: [PATCH] [SW-0] security: add is_eanbled declaration

For some reason we did not have the preboot_security_is_enabled()
declaration.
Add the declaration to security.h header file and include it in fit.c
which uses the preboot_security_is_enabled() call.

Change-Id: I1b7bb310ee1642d2f67418080c5cc11c17124e65
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/include/security.h b/include/security.h
index 7c9a567..47fd2c2 100644
--- a/include/security.h
+++ b/include/security.h
@@ -7,6 +7,9 @@
 #ifndef _HL_SECURITY_H_
 #define _HL_SECURITY_H_
 
+#include <preboot.h>
+
+bool preboot_security_is_enabled(void);
 void preboot_security_init(void);
 
 #endif /* _HL_SECURITY_H_ */
diff --git a/preboot/fit.c b/preboot/fit.c
index 3dd0ab5..e349975 100644
--- a/preboot/fit.c
+++ b/preboot/fit.c
@@ -6,6 +6,7 @@
  */
 
 #include <libfdt.h>
+#include <security.h>
 #include "common.h"
 #include "uart.h"
 #include "fit.h"
